content/en/blog/porterlb-boosts-kubernetes-on-bare-metal.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "PorterLB: Boosts Kubernetes on Bare Metal"
```

Comment:
The title looks too simple, it has no relationship with PorterLB' core characteristics.